src/pages/groupComparison/GroupComparisonStore.ts

```diff
@@ -368,6 +368,9 @@ export default class GroupComparisonStore extends ComparisonStore {
     readonly mutations = remoteData({
         await: () => [this.samples, this.mutationEnrichmentProfiles],
         invoke: async () => {
+            if (!this.activeMutationMapperGene) {
```

Comment:
this is kind of awkward.  ideally, we would never reference this if there wasn't an activeMutationMapperGene.  that's the way our code should be constructed